include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -697,7 +737,22 @@ Parameters::print_parameters_spatial_discretization(dealii::ConditionalOStream c
 
   grid.print(pcout);
 
-  print_parameter(pcout, "Polynomial degree velocity", degree_u);
+  print_parameter(pcout, "Element type", enum_to_string(spatial_discretization));
```

Comment:
This comment will cause tests to fail. Hence, you need to adjust the output of tests so that we can merge this PR.